Sources/PayPal/UIComponents/PayPalClient/PayPalClient.swift

```diff
@@ -0,0 +1,61 @@
+import UIKit
+
+#if canImport(PaymentsCore)
+import PaymentsCore
+#endif
+
+#if canImport(PayPalCheckout)
+@_implementationOnly import PayPalCheckout
+#endif
+
+public typealias PayPalCheckoutCompletion = (PayPalCheckoutResult) -> Void
+
+/// PayPal Paysheet to handle PayPal transaction
+public class PayPalClient {
+
+    private let config: CoreConfig
+    private let returnURL: String
+
+    lazy var paypalCheckoutConfig: CheckoutConfig = {
+        CheckoutConfig(
+            clientID: config.clientID,
+            returnUrl: returnURL,
+            environment: config.environment.toPayPalCheckoutEnvironment()
+        )
+    }()
+
+    public init(config: CoreConfig, returnURL: String) {
```

Comment:
We should add a docstring for this initializer, with specific info on what the returnURL is.

Reply:
This is what we had written for when the BT SDK was going to pull in NXO, if we want to pull from this language:

```
    /**
     Initializes a PayPal Checkout request.
     - Parameter payPalReturnURL: The return URL provided to the PayPal Native UI experience.
     Used as part of the authentication process to identify your application. This value should match the one set in the `Return URLs` section of your application's dashboard on your [PayPal developer account](https://developer.paypal.com)
     - Returns: A PayPal Checkout request.
     */
```